.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,43 @@
+## Description
+
+Adds [App Name] to the community train.
+
+[Brief description of what the app does]
+
+## App Information
+
+- **Upstream**: https://github.com/upstream/repo
+- **Documentation**: https://docs.upstream.com
+- **License**: MIT/etc
+- **App Version**: 2.1.0
+
+## Testing
+
+Tested locally with:
+- [ ] basic-values.yaml
+- [ ] advanced-values.yaml
+- [ ] GPU passthrough (if applicable)
```

Comment:
lets remove advanced-values, as this might hint people to start exposing any option under the sun for each app. Advanced options are usually abstracted away or left for the user via additional envs or args etc.

GPU passthrough might be ahrd to test locally, as the library assumes some groups based on the truenas itself.